ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/ConnectorUtils.java

```diff
@@ -252,4 +263,37 @@ public static SdkHttpFullRequest signRequest(
 
         return signer.sign(request, params);
     }
+
+    public static String parametersObjectExtractor(MLInput mlInput) throws IOException {
+        String mlInputString = mlInput.toXContent(XContentFactory.jsonBuilder(), ToXContent.EMPTY_PARAMS).toString();
+        ObjectMapper mapper = new ObjectMapper();
+        JsonNode mlInputObject = mapper.readTree(mlInputString);
+        return mlInputObject.get("parameters").asText();
+    }
+
+    public static void validateSchema(String schemaString, String instanceString) throws IOException {
+        System.out.println(schemaString);
```

Comment:
Remove this line, use log if need to print log

Reply:
Sure